pandas/tools/plotting.py

```diff
@@ -1134,6 +1134,31 @@ def _compute_plot_data(self):
 
         self.data = numeric_data
 
+    def _compute_plot_data_with_dt(self):
```

Comment:
Ah I misunderstood. Pls move the logic to ``PlanePlot`` as this is irrelevant from main plots.